{"name": "Проверка ошибки создания курьера: создание двух курьеров с одинаковым логином", "status": "passed", "start": 1699380922050, "stop": 1699380922424, "uuid": "8209db79-360f-4822-8b59-766e15382f78", "historyId": "1e791bea568bd8e2f9dd08c52c2ba495", "testCaseId": "1e791bea568bd8e2f9dd08c52c2ba495", "fullName": "tests.test_courier_creation.TestCourierCreation#test_courier_duplicate_login_failed_create", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_creation"}, {"name": "subSuite", "value": "TestCourierCreation"}, {"name": "host", "value": "WKS-084"}, {"name": "thread", "value": "16844-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_creation"}]}